.github/workflows/format-workflow.yml

```diff
@@ -1,9 +1,10 @@
+---
 name: Format + Docs Workflow
 on:
   push:
-    paths: ["docs/**","**.md","**.toml","**.js","**.json","**.ts"]
+    paths: ["docs/**", "**.md", "**.toml", "**.js", "**.json", "**.ts"]
```

Comment:
Could we equivalently fix this just by deleting `on push`, or does that result in other issues?

Reply:
No, that applies to the entire workflow. Ignore me.